### PR TITLE
[TD-361] Helm restructure

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

**Comment by marksou** [comment]
We need to consider how we keep the repo and docs in sync for any changes.

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md [hunk 1]
```diff
@@ -17,25 +17,177 @@ There are two main ways to install Tyk on Kubernetes: Via our Helm chart, or man
 
 ---
 
+## Installing Tyk on Kubernetes Manually
+
+We have an [archived GitHub repo](https://github.com/TykTechnologies/tyk-kubernetes) That details installing Tyk on Kubernetes manually. Feel free to try this method, but it has been superseeded by our Helm Chart and Tyk Operator offering.
```
Comment (rjohnston789) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md]: 'That' following the 'archived repo' link shouldn't be capitalised .

Superseeded should read superseded

Comment (marksou) [reply tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md]: @rjohnston789 we have decided to remove that section :-)

**Comment by letzya** [comment]
Suggest waiting for craig since I am not sure about the structure

**Comment by letzya** [comment]
@marksou I think we need to split the README from GH into a few pages. everything from TIB and down can be put on a separate page. What do you think?

**Pull request closed without merging**